target/linux/generic/files/drivers/net/phy/ar8327.c

```diff
@@ -1268,6 +1284,35 @@ ar8327_sw_set_igmp_v3(struct switch_dev *dev,
 	return 0;
 }
 
+static int
+ar8327_sw_set_port_vlan_prio(struct switch_dev *dev, const struct switch_attr *attr,
+			     struct switch_val *val)
+{
+	struct ar8xxx_priv *priv = swdev_to_ar8xxx(dev);
+	int port = val->port_vlan;
+
+	if (port >= dev->ports)
+		return -EINVAL;
+	if (port == 0 || port == 6)
+		return -EOPNOTSUPP;
+
+        priv->port_vlan_prio[port] = val->value.i;
```

Comment:
looks like a stray space at the start of the line.

Reply:
amended